flathub.json

```diff
@@ -1,3 +0,0 @@
-{
-  "only-arches": ["x86_64"]
```

Comment:
I think this recipe will only work on x86_64 and aarch64, so we should put those two in here, don't you think?

Reply:
this are the only two arches built with that version of the runtime, so there is no point.

Reply:
I thought so but couldn't find a source for it. Alright then, we can leave that as is, might as well become a reminder in case other arches will be added in the future

Reply:
> I thought so but couldn't find a source for it. 

Simple, the build on PR is automatic and it was successful. It only lists x86_64 and aarch64.